src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -21,6 +20,7 @@ internal class ValidationParameters
         private string _nameClaimType = ClaimsIdentity.DefaultNameClaimType;
         private string _roleClaimType = ClaimsIdentity.DefaultRoleClaimType;
         private Dictionary<string, object> _instancePropertyBag;
+        private IList<string> _validIssuers = [];
```

Comment:
Leave these _validIssuers and _validTokenTypes null.
Use the pattern shown in the Get below.